src/modifiers/modifiers.py

```diff
@@ -8,12 +8,20 @@
     "bound": "excludeInterior",
 }
 
+head_tail = {
+    "head": "head",
+    "tail": "tail",
```

Comment:
Possibly:

```suggestion
    "head": "extendThroughStartOf",
    "tail": "extendThroughEndOf",
```

I don't use "line" here to allow for eg "tail funk" in the future. Could see making slightly more verbose to eg `extendThroughStartOfScope`